backend/src/main/java/com/jos/spotifyclone/controller/SearchController.java

```diff
@@ -15,10 +16,45 @@
     SpotifyConnect spotifyConnect;
 
     //TODO ${ARTIST_NAME_HERE} needs value storing elsewhere where this controller can access the search term to return searched for artist data
-    //http://localhost:8080/api/search/artist/?artist=drake
+    //http://localhost:8080/api/search/artist/?id=drake
```

Comment:
We should change this so it is /artist?id=drake as the extra slash is not needed. But something for another PR

Reply:
I would argue id is the wrong word to use here also. Id normally means something numeric but I don't have a better idea right now...